internal/transport/http_util.go

```diff
@@ -603,6 +603,11 @@ func newFramer(conn net.Conn, writeBufferSize, readBufferSize int, maxHeaderList
 // parseDialTarget returns the network and address to pass to dialer.
 func parseDialTarget(target string) (string, string) {
 	net := "tcp"
+	if strings.HasPrefix(target, "unix-abstract:") {
+		// handle unix-abstract:addr which will fail with url.Parse
+		// and need prepend "\x00" to the parsed address
+		return "unix", "\x00" + strings.SplitN(target, ":", 2)[1]
+	}
```

Comment:
This should not be necessary.  We only do this for "unix" for backward compatibility reasons.  I.e. we don't need to support `passthrough:///unix-abstract:path`.

Reply:
Understood, removed along with the test cases